exercises/bank-account/instructions.md

```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+Your task is to implement bank accounts supporting opening/closing, withdrawals, and deposits of money.
+
+As bank accounts can be accessed in many different ways (internet, mobile phones, automatic charges), your bank software must allow accounts to be safely accessed from multiple threads/processes (terminology depends on your programming language) in parallel.
+For example, there may be many deposits and withdrawals occurring in parallel; you need to ensure there is no [race conditions][helpful link] between when you read the account balance and set the new balance.
```

Comment:
`you need to ensure there is no [race conditions][helpful link]`

Does anyone have a helpful link for race conditions?

* https://en.wikipedia.org/wiki/Race_condition#In_software
* https://www.techtarget.com/searchstorage/definition/race-condition
* https://devopedia.org/race-condition-software

Reply:
I liked this one for its refreshing lack of ads: https://startup-house.com/glossary/race-condition.
But the devopedia one is nice with its diagrams.

Reply:
I like the wikipedia link best, with clear text and the table it shows is very helpful I feel.